features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/resources/DefaultImageWireframeHelper.kt

```diff
@@ -94,31 +92,35 @@ internal class DefaultImageWireframeHelper(
         val drawableWidthDp = width.densityNormalized(density).toLong()
         val drawableHeightDp = height.densityNormalized(density).toLong()
 
-        val imageWireframe = MobileSegment.Wireframe.ImageWireframe(
-            id = id,
-            x,
-            y,
-            width = drawableWidthDp,
-            height = drawableHeightDp,
-            shapeStyle = shapeStyle,
-            border = border,
-            clip = clipping,
-            mimeType = mimeType,
-            isEmpty = true
-        )
+        val imageWireframe =
+            MobileSegment.Wireframe.ImageWireframe(
+                id = id,
+                x,
+                y,
+                width = drawableWidthDp,
+                height = drawableHeightDp,
+                shapeStyle = shapeStyle,
+                border = border,
+                clip = clipping,
+                isEmpty = true
```

Comment:
why we don't need `mimeType` anymore?

Reply:
We're sending the type in the [resource post](https://github.com/DataDog/dd-sdk-android/blob/jmoskovich/rum-2708/new-refactor/features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/net/ResourceRequestBodyFactory.kt#L197), therefore we don't need it anymore in the wireframe